docs/src/pages/api/01_authentication.md

```diff
@@ -29,6 +38,9 @@ const appOctokit = new Octokit({
   auth: {
     id: 123,
     privateKey: process.env.PRIVATE_KEY,
+    // optional: this will make appOctokit authenticate as app (JWT)
+    //           or installation (access token), depending on the request URL
+    installationId: 123,
```

Comment:
yes, this part would definitely have helped for sure!

Reply:
thanks for the review 👍🏼